### ISSUE TITLE
Page numbers appear with the prefix "p." how do I stop this?

### ISSUE THREAD

**Issue by clarkie1977** [retorquere/zotero-better-bibtex#1262]
Hi there, 

I know nothing about code or half the stuff probably required. I have Better-Bibtex working with Zotero. I am on a Mac and using an Apple script called Zotpick with pandoc to add ciations to Ulysses. It works really well but the added citekeys need a little manual amending. An example of how it exports to Ulysses is: 

@dillonBlackstoneEmergencyPlanning2014, p. 1

I want to make it export like this:

[@dillonBlackstoneEmergencyPlanning2014, 1]

Adding the brackets is not so much of a problems therfore the most important part is loosing the "p." as when it is processed to my citation style they carry over. 

Any ideas or solutions will be much appriciated. I havent a clue about this sort of thing. 

Thanks.

**Comment by retorquere** [comment]
You can ignore label-gun. I'll look at your request later today.

**Comment by clarkie1977** [comment]
Hi there,

Thank you for looking later. I didn't catch the label-gun message. I did get an email last night saying I hadn't uploaded or included something. I apologise if I've used the wrong space/area/forum. I'm on an MSc assignment deadline and just rushing through anything not directly related. Once I've submitted my paper I'll try and bring my self up to speed with these sort of things. I think in this day an age learning a little code would be a good idea (if that's even possible of course). Thanks again, it will be a huge help toward my assignment. Cheers!
